libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -48,12 +48,21 @@
 
 static NSMutableDictionary *analyticsManagerList = nil;
 
+@interface SFSDKAnalyticsTransformPublisherPair : NSObject
```

Comment:
The `NSCopying` thing simply to support the `NSDictionary` bugged me—internal details driving external requirements.  So I just created an internal structure to manage the relationship instead.